daemon/inspect.go

```diff
@@ -58,6 +59,13 @@ func (daemon *Daemon) getInspectData(container *Container) (*types.ContainerJSON
 		FinishedAt: container.State.FinishedAt.Format(time.RFC3339Nano),
 	}
 
+	sizeRwStr, sizeRootFsStr := "N/A", "N/A"
+	if size {
+		sizeRw, sizeRootFs := container.GetSize()
+		sizeRwStr = units.HumanSize(float64(sizeRw))
+		sizeRootFsStr = units.HumanSize(float64(sizeRootFs))
```

Comment:
I agree with @thaJeztah. Those values should be raw, without human transformations.